sdk/maps/maps-weather-rest/src/mapsWeather.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We should sync the client file name, using capital "M". Please help to rename it to `MapsWeather.ts`